aws/lambda/node/README.md

```diff
@@ -0,0 +1,21 @@
+# Node Lambdas
+
+## Running Locally
+
+### Setup
+Install [aws-sam-cli](https://github.com/awslabs/aws-sam-cli/blob/develop/docs/installation.rst)
```

Comment:
Would it be better to use a link to either `master` or a specific version instead of `develop`?